### PR TITLE
ARXIVNG-1968: four column footer

### PR DESCRIPTION
**Pull request by mhl10**

### CODE REVIEW

File: browse/templates/base.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   {%- block head -%}
   <title>{% block title %}{% endblock %}</title>
   <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
-  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/arXiv-20190208.css?v=20190208" />
+  <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/arXiv-ARXIVNG-1918.css?v=20190307" />
```
Comment (erickpeirson) [inline browse/templates/base.html]: Is ``arXiv-ARXIVNG-1918.css`` right? Does it exist?

Comment (mhl10) [reply browse/templates/base.html]: Thanks for pointing this out. It does exist. Once I deploy browse and confirm all is OK, I'll update the "master" copy arXiv.css in the CDN and point browse to that. Want to minimize issues due to caching.

File: browse/templates/base.html [hunk 2]
```diff
@@ -38,10 +38,17 @@
   <script language="javascript" src="//code.jquery.com/jquery-latest.min.js"></script>
   <script language="javascript" src="//static.arxiv.org/js/user_banner.js"></script>
   {% endif -%}
+  <style type="text/css">
+    @media screen and (max-width: 1023px) {
```
Comment (eawoods) [inline browse/templates/base.html]: hold for update to the width on this number, I have to reconcile this with other pages (it forces columns on footer)

Comment (eawoods) [reply browse/templates/base.html]: yep, this rule that I added should now be removed. It's superseded in the arxiv-ARXIVNG-1918.css stylesheet.

**Pull request merged**